ui/src/app/views/home-page/component.ts

```diff
@@ -50,9 +51,9 @@ export class HomePageComponent implements OnInit {
   ) {}
   ngOnInit(): void {
     this.workspacesLoading = true;
-    this.workspacesService
-        .getWorkspaces()
-        .retry(2)
+
+    retryApi(this.workspacesService
```

Comment:
How about setting the default retries to 3, and then we only have to pass in a value if there's some specifically flaky endpoint that needs more?

Reply:
Agreed. I had considered doing a default value, and three seems a reasonable number.